notify/webhook/webhook.go

```diff
@@ -106,5 +107,16 @@ func (n *Notifier) Notify(ctx context.Context, alerts ...*types.Alert) (bool, er
 	}
 	notify.Drain(resp)
```

Comment:
we have a bug here I believe :)

```suggestion
	defer notify.Drain(resp)
```